metadata-ingestion/src/datahub/ingestion/source/snowflake/snowflake_report.py

```diff
@@ -1,11 +1,13 @@
-from typing import MutableSet, Optional
+from dataclasses import dataclass, field
+from typing import Dict, MutableSet, Optional
 
 from datahub.ingestion.source.snowflake.constants import SnowflakeEdition
 from datahub.ingestion.source.sql.sql_generic_profiler import ProfilingSqlReport
 from datahub.ingestion.source_report.sql.snowflake import SnowflakeReport
 from datahub.ingestion.source_report.usage.snowflake_usage import SnowflakeUsageReport
 
 
+@dataclass
```

Comment:
Pretty sure we want this on every class, even though it inherits from a dataclass?